src/main/java/com/unstoppabledomains/resolution/DomainResolution.java

```diff
@@ -0,0 +1,119 @@
+package com.unstoppabledomains.resolution;
+
+import com.unstoppabledomains.exceptions.NamingServiceException;
+
+public interface DomainResolution {
+
+    /**
+     * Resolves domain for a specific ticker address
+     *
+     * @param domain - domain name such as "brad.crypto"
+     * @param ticker - coin ticker such as ETH
+     * @return address in hex-string format
+     * @throws NamingServiceException - if tld is not recognized or there record for such currency is not presented
```

Comment:
I'm not sure I understand what this sentence means

Reply:
I just moved this javadoc from another place, so let me clarify with the author and get back to this one 👌🏼

Reply:
@mvlabat please, check it out. Does the new version make more sense now?